lib/tools/mp4-inspector.js

```diff
@@ -529,11 +529,14 @@ var
         defaultSampleDurationPresent = result.flags[2] & 0x08,
         defaultSampleSizePresent = result.flags[2] & 0x10,
         defaultSampleFlagsPresent = result.flags[2] & 0x20,
+        durationIsEmpty = result.flags[0] & 0x010000,
+        defaultBaseIsMoof =  result.flags[0] & 0x020000,
         i;
 
       i = 8;
       if (baseDataOffsetPresent) {
         i += 4; // truncate top 4 bytes
+        // FIXME: should we read the full 64 bits?
```

Comment:
JS can't handle 64-bit numbers natively, so we'd have to add a dependency for some package that can, but that's a good question :thinking: